bitcoin/src/crypto/sighash.rs

```diff
@@ -463,12 +463,26 @@ impl EcdsaSighashType {
         }
     }
 
+    /// Converts [`EcdsaSighashType`] to a `u8` sighash flag.
+    ///
+    /// The returned value is guaranteed to be a valid according to standardness rules.
+    pub fn to_u8(self) -> u8 { self as u8 }
+
     /// Converts [`EcdsaSighashType`] to a `u32` sighash flag.
     ///
     /// The returned value is guaranteed to be a valid according to standardness rules.
     pub fn to_u32(self) -> u32 { self as u32 }
```

Comment:
I wonder if it makes sense to have this public since it's only used in signing code. But I guess we don't have to remove it either.

Reply:
FTR this has been removed.